### DISCUSSION TITLE
Enemy submarine still press player submarine to the bottom in random encounters

### DISCUSSION THREAD

**Discussion by Instant2116** [FakeFishGames/Barotrauma#13855] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Once all hostile NPC are dead enemy submarine supposed to become lighter, so player submarine can still ascend even been under enemy vessel, **but It doesn't work in random encounters when enemy submarine spawns not by quest, but by hostile reputation with faction.**

### Reproduction steps

1. start singleplayer campaign
2. make 'hostile' reputation with one of the main faction
3. cause "you can't swim here" event, (when enemy vessel is spawned)
4. destroy enemy while been under enemy submarine
5. try to ascend (enemy submarine press you towards the bottom)

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.4.5.0 (Blood in the Water Update, hotfix 1)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I unfortunately could not find a way to reproduce the issue, "lightening" the sub on top seems to work as expected. I suspect there's some other factor that's preventing it from working properly in your case, but I'm not entirely sure what that could be. A save file where this occurs consistently, or some more details on the issue (which submarine are you playing on, which submarine was the enemy team using, which faction was attacking you...?) might help us diagnose this. Sorry for the inconvenience!
